### ISSUE TITLE
Make is:dupe ignore items with Junk tag (or any tag?)

### ISSUE THREAD

**Issue by TheGokki** [DestinyItemManager/DIM#8747]
Title. TLDR at the bottom.

I have a bunch of items that are tagged as `Junk` or `Infuse` for me to use as infusion fodder and would be dismantled otherwise. They are not considered part of my weapon porfolio so when im searching for dupes i would prefer if such junk weapons (or armor) would be ignored when cheking for duplicates.

Example:

I have a Krait with good perks. I also have two additional Kraits with red borders sitting there as bank for Resonant Elements (i'm capped). They are tagged as `Junk`.

I also have a couple of pinnacle drops that I am holding onto for infusion, they are otherwise useless and are tagged as `Infuse`.

Both of those examples are weapons i would prefer ignored when checking for duplicates, i know what they are and have already tagged and sorted.

TLDR:
Suggestion: have a toggle option to make `is:dupe` ignore each individual tag.
Alternative suggestion: make `is:dupe -tag:junk` interact to only count non-Junk items as duplicate, though typing this out in the search every time is kind of a pain.

**Comment by itanne99** [comment]
Why not make `is:dupe -tag:junk` a favorited query?
Ex: I have `is:dupelower -is:crafted -is:wishlist -is:deepsight light:<1570 -is:exotic` as favorited and makes life pretty easy when it comes to semi-clearing my inventory

**Comment by TheGokki** [comment]
> Why not make `is:dupe -tag:junk` a favorited query? Ex: I have `is:dupelower -is:crafted -is:wishlist -is:deepsight light:<1570 -is:exotic` as favorited and makes life pretty easy when it comes to semi-clearing my inventory

That query doesn't actually work the way i've described, this Issue is for that.

That said - I've "run out of space" for favorited query, there are no buttons i can summon queries from, the drop down menu is constantly changing the favorited queries and sometimes doesn't show the four i have already. I would love if each query would get a small button inside the search bar on the right where i could just click it to summon the query. Alas, none of this is possible, currently.

**Comment by TheGokki** [comment]
> Why not make `is:dupe -tag:junk` a favorited query? Ex: I have `is:dupelower -is:crafted -is:wishlist -is:deepsight light:<1570 -is:exotic` as favorited and makes life pretty easy when it comes to semi-clearing my inventory

Favorite queries don't really work well, the favorites are just part of the autocomplete dropdown while typing so there's an effective maximum of them, not to mention they are constantly jumbled or don't even show up. I would need some buttons, say inside the search window, where i can click the favorited queries.

I've made this suggestion before but the responce i got was "not enough space in the UI" or something.

**Comment by delphiactual** [comment]
filters are not aware of one another.

**Comment by bhollis** [comment]
You can have as many favorite queries as you want. You just need to search for them and they'll come up.

**Comment by TheGokki** [comment]
> filters are not aware of one another.

Yes, which is precisely why i've created this Issue x) Ideally there's an option for it so i can just type `is:dupe`, but if not then having filters be intra-aware.

> You can have as many favorite queries as you want. You just need to search for them and they'll come up.

if i have to type it makes them useless imo.

**Comment by bhollis** [comment]
We unfortunately don't have any plans to make filters that interact with / are conditional on each other, nor do we want to add config options for filters or add a bunch of different filters. My suggestion is to use the Compare or Organizer tools to decide which items to keep.
